include/oneapi/dpl/iterator

```diff
@@ -34,19 +34,17 @@ namespace dpl
 using ::std::advance;
 using ::std::begin;
 using ::std::bidirectional_iterator_tag;
-#if __cplusplus > 201103L
 using ::std::cbegin;
 using ::std::cend;
 using ::std::crbegin;
 using ::std::crend;
+using ::std::distance;
 using ::std::make_reverse_iterator;
 using ::std::rbegin;
 using ::std::rend;
-#endif // __cplusplus > 201103L
-using ::std::distance;
 #if __cplusplus > 201402L
-using ::std::empty;
-using ::std::size;
+using ::std::empty;         // KSATODO: leve as is: https://en.cppreference.com/w/cpp/iterator/empty - since С++17
+using ::std::size;          // KSATODO: leve as is: https://en.cppreference.com/w/cpp/iterator/size - since С++17
 #endif // __cplusplus > 201402L
```

Comment:
Why keeping the "greater than C++14" condition check?

Reply:
Fixed: removed support of C++14 too